tests/kernel/fatal/exception/src/main.c

```diff
@@ -90,6 +90,35 @@ void entry_cpu_exception(void *p1, void *p2, void *p3)
 	rv = TC_FAIL;
 }
 
+void entry_cpu_exception_divide_zero(void *p1, void *p2, void *p3)
```

Comment:
According to below question, would it be better to add some descriptions for this subroutine? thanks!